src/ObjectMultiplex.ts

```diff
@@ -0,0 +1,93 @@
+import { Duplex } from 'readable-stream';
+import eos from 'end-of-stream';
```

Comment:
This import used to be called `endOfStream`, which is more readable and therefore superior.

Reply:
I could not import `endOfStream` from the package `end-of-stream`. `@types/end-of-stream` is exporting `eos`